getting_started/scripting/gdscript/gdscript_styleguide.rst

```diff
@@ -418,6 +418,39 @@ characters in a given string. See the examples below:
     # Both quote styles would require 2 escapes; prefer double quotes if it's a tie.
     print("'hello' \"world\"")
 
+Numbers
+~~~~~~~
+
+Use lowercase for letters in hexadecimal numbers, as their lower height makes
+the number easier to read.
+
+**Good**::
+
+    var hex_number = 0xfb8c0b
+
+**Bad**::
+
+    var hex_number = 0xFB8C0B
+
+Take advantage of GDScript's underscores in litereals to make large numbers more
+readable.
+
+**Good**::
+
+    var large_number = 1_234_567_890
+    var large_hex_number = 0xffff_f8f8_0000
+    var large_bin_number = 0b1101_0010_1010
+    # Numbers lower than 1000000 generally don't need separators.
+    var small_number = 12345
+
+**Bad**::
+
+    var large_number = 1_234_567_890
```

Comment:
I think you forgot to remove the underscores here. :)
```suggestion
    var large_number = 1234567890
```

Reply:
Good catch! Fixed.